src/twisted/python/test/test_release.py

```diff
@@ -453,7 +449,7 @@ def test_build(self):
         )
         self.assertIn(
             '<a href="{}">{}</a>'.format(projectURL, projectName),
-            indexPath.getContent(),
+            indexPath.getContent().decode("ascii"),
```

Comment:
Do these need to be "ascii"?
```suggestion
            indexPath.getContent().decode(),
```

Reply:
well.. I just want to convert from bytes to str... and I want to go with a fixed encoding which will never fail.

For example utf-16 decoding will fail for an odd number of bytes.

As a best practice, I never used the default encoding.

Let me know if there is a better way of converting from bytes to ascii str.

Reply:
the recommended way from pyupgrade is to just rely on the default - as it will always be `utf8`

https://github.com/asottile/pyupgrade#encodeutf-8

Reply:
Thanks. True stdlib docs - https://docs.python.org/3/library/stdtypes.html#bytes.decode

I am still a python2 developer and I am not used to burn py2 bridges :) but I will update this. Happy to see a fixed default in py3.

]